docs/pages/workflow/logging.md

```diff
@@ -2,11 +2,14 @@
 title: Viewing logs
 ---
 
+import { Terminal } from '~/ui/components/Snippet';
+
 Writing to the logs in an Expo app works just like in the browser: use `console.log`, `console.warn` and `console.error`. Note: we don't currently support `console.table` outside of remote debugging mode.
 
 ## Recommended: View logs with Expo tools
 
-When you open an app that is being served from Expo CLI, the app will send logs over to the server and make them conveniently available to you. This means that you don't need to even have your device connected to your computer to see the logs -- in fact, if someone opens the app from the other side of the world you can still see your app's logs from their device.
+When you open an app that is being served from Expo CLI, the app will send logs over to the server and make them conveniently available to you. 
+This means that you don't need to even have your device connected to your computer to see the logs -- in fact, if someone opens the app from the other side of the world you can still see your app's logs from their device.
```

Comment:
Do you intend to add a line break here? If yes, I think you need to leave an extra line in between them.

Reply:
This change is just for the editors readability (it's cumbersome to work with very long lines), this do not affect how the text is rendered on the website and it was my intent here.

Reply:
Ah okay, got it! Thanks!